### PR TITLE
Kh360-react 444 my evaluation list mobile view

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
[My Evaluations List - adjust to make it mobile responsive#444](https://github.com/nerubia/kh360-react/issues/444)

### CODE REVIEW

**Review by emmaacha — APPROVED** [review]

File: src/features/my-evaluations/my-evaluations-list.tsx [hunk 1]
```diff
@@ -8,9 +8,10 @@ import { formatDateRange } from "../../utils/format-date"
 import { Spinner } from "../../components/ui/spinner/spinner"
 import { getByTemplateType } from "../../redux/slices/email-template-slice"
 import { ScoreRange } from "../../components/shared/score-range/score-range"
-
+import useMobileView from "../../hooks/use-mobile-view"
```
Comment (jatnerubia) [inline src/features/my-evaluations/my-evaluations-list.tsx:11]: add new line here

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

File: src/features/my-evaluations/my-evaluations-list.tsx (hunk 1 above)
Comment (chadiegilnerubia) [reply src/features/my-evaluations/my-evaluations-list.tsx:11]: done :3 hehe

**Review by jatnerubia — APPROVED** [review]
Cool

**Pull request merged**